radar-prmt-ios/AppDelegate.swift

```diff
@@ -29,21 +29,33 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     let controlQueue: SchedulerType = SerialDispatchQueueScheduler(qos: .background)
     var lastServerStatus = BehaviorSubject<KafkaEvent>(value: .disconnected(Date()))
     lazy var authController = { AuthController(config: config) }()
-
+    var lastAppState: AppState = AppState.inactive
```

Comment:
Could this be made into a BehaviorSubject instead? Reason being: the latestConfig subject now will not be updated on app state changes.